### DISCUSSION TITLE
Store initialization error occurs when the page switches and go back in chrome,Firefox is fine

### DISCUSSION THREAD

**Discussion by frankdiw** [mobxjs/mobx#3512] [Q&A]
When the page jumps to another page, refresh the new page, click go back, the store initialization does not take effect，But if not refresh,click go back, initialization was work good. who can help me resolve problem.thanks
<img width="637" alt="image" src="https://user-images.githubusercontent.com/17613480/187817615-a447f0dc-6073-427b-8355-f8c3df75d6d0.png">

**Comment by kubk** [comment]
Unfortunately it's not enough to say what exactly is wrong so could you create a minimal reproducible codesandbox first? 

https://codesandbox.io/s/minimal-mobx-react-project-ppgml?file=/index.js

**Reply by frankdiw** [reply]
sorry,I can't reproduce, I provide some picture for this error,
<img width="366" alt="image" src="https://user-images.githubusercontent.com/17613480/187863254-604c71b7-5047-4aa6-a863-4912340ff91a.png">
above picture is a right.
<img width="345" alt="image" src="https://user-images.githubusercontent.com/17613480/187863480-ba658d5b-0e34-41b4-bd17-aef25cb7a1c9.png">
above picture is error,  useLocalObservable received the same value return different result。

**Comment by urugator** [comment]
What does `createStore` return? `useLocalObservable` expects callback (factory function).
https://github.com/mobxjs/mobx/blob/f0e066f427d573cbef4b92f3310eb069c3aca205/packages/mobx-react-lite/src/useLocalObservable.ts#L5

**Reply by frankdiw** [reply]
<img width="628" alt="image" src="https://user-images.githubusercontent.com/17613480/188039779-0d31193c-d7e5-40b2-9979-2bbfec84f103.png">
createStore return is correct
